vulkaninfo/vulkaninfo.h

```diff
@@ -74,6 +74,7 @@
 bool human_readable_output = true;
 bool html_output = false;
 bool json_output = false;
+bool vkconfig_output = false;
```

Comment:
Should these bools be private inside main()?  They only seem to have utility between arg parsing and construction of the Printer instance, which all happens in main()

Reply:
This is because a macro used the `human_readable_output` bool in the header file itself. I could move the json/html/vkconfig bools elsewhere, but that only makes the placement of `human_readable_output` even more confusing than declaring them all together.